### ISSUE TITLE
Add CLI Usage

### ISSUE THREAD

**Issue by zeke** [zeit/next.js#80]
I expected `next --help` to be a thing:

```
❯ next --help
> Ready on http://localhost:3000
> Couldn't find a `pages` directory. Please create one under the project root
```

`next build` is currently undocumented. My goal was to discover its purpose without reading the code. 😛

**Comment by timneutkens** [comment]
Fixed in #423 `next build --help` `next --help`

**Comment by zeke** [comment]
🎉
